packages/framework-integration-tests/integration/provider-unaware/load/entities-are-correctly-reduced.load.ts

```diff
@@ -0,0 +1,77 @@
+import { ApolloClient } from 'apollo-client'
+import { NormalizedCacheObject } from 'apollo-cache-inmemory'
+import { internet, random } from 'faker'
+import gql from 'graphql-tag'
+
+import { waitForIt } from '../../helper/sleep'
+import { applicationUnderTest, scriptExecutor } from './setup'
+
+describe('Data consistency on entities', () => {
+  let client: ApolloClient<NormalizedCacheObject>
+  let token: string
+
+  before(async () => {
+    const userEmail = internet.email()
+    token = applicationUnderTest.token.forUser(userEmail, 'Admin')
+    client = applicationUnderTest.graphql.client(token)
+  })
+
+  context('with 200 products ready to be stocked', () => {
+    const idPrefix = random.alpha({ count: 10 })
+    const destinationWarehouse = 'GC'
+    const numberOfProducts = 1000
+    let productIDs: Array<string>
+
+    before(async () => {
+      productIDs = []
+      for (let i = 0; i < numberOfProducts; i++) {
+        productIDs[i] = idPrefix + random.uuid()
+      }
+    })
+
+    it('adds stock to all of them with many events without corrupting data', async () => {
+      const durationWarmup = 10
+      const arrivalRateWarmup = 200
+      const durationBurst = 10
+      const arrivalRateBurst = 2000
+      const expectedStock = durationWarmup * arrivalRateWarmup + durationBurst * arrivalRateBurst
+      await scriptExecutor.executeScript('move-product-stock.yml', {
+        variables: { token, productID: productIDs, destinationWarehouse },
+        phases: [
+          {
+            duration: durationWarmup,
+            arrivalRate: arrivalRateWarmup,
+          },
+          {
+            duration: durationBurst,
+            arrivalRate: arrivalRateBurst,
+          },
+        ],
+      })
```

Comment:
We send those many events in two batches. The first one has a slower rate (only 200 events per second) to give time to lambdas to warm up. The second one will send 2000 events per second

If we don't do this, we will probably get throttling exceptions from the lambda service